cosmo_tester/test_suites/manager_of_managers/regional_cluster.py

```diff
@@ -0,0 +1,984 @@
+                 '-t', self.tenant, '-vv'])
+        except Exception as e:
+            if 'if there is a running system-wide' in e.message:
+                self.logger.error('Error on deployment execution: %s', e)
+                self.logger.info('Listing executions..')
+                self.cfy.executions.list(['-d', self.deployment_id])
+                self.cfy.executions.list(['--include-system-workflows'])
+            raise
+
+    def execute_hello_world_workflow(self, workflow_id):
+        self.logger.info(
+            'Executing workflow {0} on deployment {1} '
+            'on a Regional cluster...'.format(
+                workflow_id,
+                constants.HELLO_WORLD_DEP)
+        )
+        workflow_params = {
+            'workflow_id': workflow_id,
+            'deployment_id': constants.HELLO_WORLD_DEP,
+            'tenant_name': constants.TENANT_1
+        }
+
+        self.cfy.executions.start([
+            'execute_workflow',
+            '-d', self.deployment_id,
+            '-p', json.dumps(workflow_params)
+        ])
+        self.logger.info(
+            'Successfully executed workflow {0} on deployment {1} '
+            'on a Regional cluster'.format(
+                workflow_id,
+                constants.HELLO_WORLD_DEP)
+        )
+
+
+class FixedIpRegionalCluster(AbstractRegionalCluster):
+    TRANSFER_AGENTS = False
+    RESOURCE_POOL1 = [
+        {
+            'ip_address': '10.0.0.50',
+            'hostname': 'db_worker_'
+        },
+        {
+            'ip_address': '10.0.0.51',
+            'hostname': 'db_worker_2'
+        }
+    ]
+    RESOURCE_POOL2 = [
+        {
+            'ip_address': '10.0.0.52',
+            'hostname': 'queue_worker_1'
+        },
+        {
+            'ip_address': '10.0.0.53',
+            'hostname': 'queue_worker_2'
+        }
+    ]
+    RESOURCE_POOL3 = [
+        {
+            'ip_address': '10.0.0.54',
+            'hostname': 'seed_worker_1'
+        },
+        {
+            'ip_address': '10.0.0.55',
+            'hostname': 'seed_worker_2'
+        }
+    ]
+    RESOURCE_POOL4 = [
+        {
+            'ip_address': '10.0.0.56',
+            'hostname': 'additional_worker_1'
+        },
+        {
+            'ip_address': '10.0.0.57',
+            'hostname': 'additional_worker_2'
+        }
+    ]
+
+    @property
+    def network_inputs(self):
+        return {
+            # Only relevant when working with the Private Fixed IP paradigm.
+            # See more in private_fixed_ip.yaml
+            'queue-infrastructure--'
+            'resource_pool': self.RESOURCE_POOL1,
+            'database-infrastructure--'
+            'resource_pool': self.RESOURCE_POOL2,
+            'seed-worker-infrastructure--'
+            'resource_pool': self.RESOURCE_POOL3,
+            'additional-workers-infrastructure--'
+            'resource_pool': self.RESOURCE_POOL4,
+        }
+
+    @property
+    def inputs(self):
```

Comment:
Can we use the inputs from 'AbstractRegionalCluster'?

Reply:
Nope.